### DISCUSSION TITLE
hatch with no borders

### DISCUSSION THREAD

**Discussion by RauchLukas** [mozman/ezdxf#691] [General]
Hey folks, 

I am new to ezdxf, so go easy on me. 

I was wondering if you can create a hatch without the border from the polyline getting plottet? 
from my understanding, hatches are something like fillings. So deleting the Polyline should keep the hach in place. At lest, that is what it dose in AutoCAD. 

Can i do the same thin with ezdxf?
![hatch](https://user-images.githubusercontent.com/57633463/171234313-6898e0bb-27b3-41f7-a8fe-a14f34d75718.png)

**Comment by mozman** [comment]
The `HATCH` entity does not have a border line, so if you create a `HATCH` by `ezdxf` it has no border line. If a `HATCH` in an existing DXF file has a border line, this border line is a separated DXF entity like a `LWPOLYLINE` which you have to delete to remove the border line but there is no connection or link between the `HATCH` and the border line entity, so finding this border line entity is not a trivial task.

**Reply by RauchLukas** [reply]
Thanks @mozman for the quick reply. 

Checking the actual dxf output confirms that.
I was miss-leaded by the `RenderContext` output, which always creates the borders around my hatch. 

```python
from ezdxf.addons.drawing import RenderContext, Frontend
from ezdxf.addons.drawing.matplotlib import MatplotlibBackend

def render_png(doc, path='view.png'):
    fig = plt.figure()
    ax = fig.add_axes([0, 0, 1, 1])
    ctx = RenderContext(doc)
    out = MatplotlibBackend(ax)
    Frontend(ctx, out).draw_layout(doc.modelspace(), finalize=True)
    fig.savefig(path, dpi=300)
    print("rendered to: ", path)
```

<img src="https://user-images.githubusercontent.com/57633463/171347627-35718e99-6225-40f9-8aa8-7aa3af550b34.png" width="400" height="400">



I guess there is no way to avoid rendering the borders with Matplotlib Pipeline?
